src/app/screens/connectors/NewWallet/index.tsx

```diff
@@ -63,24 +65,22 @@ export default function NewWallet() {
               data
             )}`
           );
+          setLoading(false);
         }
       })
       .catch((e) => {
         console.error(e);
         toast.error(
           `${t("pre_connect.errors.create_wallet_error")} ${e.message}`
         );
-      })
-      .finally(() => {
         setLoading(false);
       });
   }
 
-  async function next(event: React.FormEvent<HTMLFormElement>) {
+  async function next(lndhub: LNDHubCreateResponse) {
```

Comment:
Would it be easier to understand if we call this i.e. `postSignUp` instead of next?

Reply:
it was next before, so I did not change it. do you care, should I change it?

Reply:
No, not blocking. I'll keep this in mind.